### PR TITLE
Created Script for the classmate v1.5.1 and added build_info.json

### PR DESCRIPTION
**Pull request by pratibh-ibm**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: c/classmate/classmate_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : java-classmate
+# Version       : 1.5.1
```
Comment (prachikurade1) [inline c/classmate/classmate_ubi_9.3.sh:5]: Update version to match the version in build info.json

**Review by prachikurade1 — CHANGES_REQUESTED** [review]
Additionally the folder, script names should match package name so it will be j/java-classmate, and java-classmate_ubi_9.3

**Comment by pratibh-ibm** [comment]
Previous version for the package were built in this folder, so didn't create new java-classmate folder.

File: c/classmate/build_info.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "package_name": "java-classmate",
+    "github_url": "https://github.com/FasterXML/java-classmate",
+    "version": "classmate-1.5.1",
+    "default_branch": "master",
+    "build_script": "classmate_ubi_9.3.sh",
+    "package_dir": "c/classmate",
+    "validate_build_script": true,
+    "use_non_root_user": false,
+    "docker_build": false,
+    "1.*.*": {
```
Comment (prachikurade1) [inline c/classmate/build_info.json:12]: Need to fix the version regex as discussed.

**Review by prachikurade1 — CHANGES_REQUESTED** [review]

**Review by prachikurade1 — APPROVED** [review]
Approved

**Pull request merged**